plugins/httpapi/eos.py

```diff
@@ -76,7 +76,7 @@ def send_request(self, data, **message_kwargs):
             data.insert(0, {"cmd": "enable", "input": self._become_pass})
 
         output = message_kwargs.get("output") or "text"
-        version = message_kwargs.get("version") or 1
+        version = message_kwargs.get("version") or "latest"
```

Comment:
@GomathiselviS @Qalthos - We are proposing to change the default behavior of httpapi transport to match the default behavior of network_cli. This could potentially be a breaking change for customers. Please let us know if any documentation/release notes need to be updated to flag this change.